src/components/InlineSvg/InlineSvg.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState, useEffect, useRef } from 'react';
+import { Icon, IconName } from '../Icon';
+import { Skeleton, SkeletonVariant } from '../Skeleton';
+
+import { InlineSvgProps } from './InlineSvg.types';
+
+export const InlineSvg = ({
+  classNames,
+  height,
+  hideBrokenIcon = false,
+  showSkeleton = false,
+  skeletonVariant = SkeletonVariant.Rounded,
+  url,
+  width,
+}: InlineSvgProps) => {
+  const [isLoading, setIsLoading] = useState(true);
+  const [hasError, setHasError] = useState(false);
+  const svgRef = useRef<HTMLDivElement>(null);
```

Comment:
If you decide to forwardRef this component and make the svg div the ref to forward, you may useMergedRef from hooks to keep this pattern of internal ref too.

Reply:
I'll just put the forward ref on the parent wrapping div, but thanks for this tip